src/ekore/anomalous_dimensions/unpolarized/time_like/as1.py

```diff
@@ -0,0 +1,175 @@
+"""The unpolarized LO Altarelli-Parisi splitting kernels."""
+
+import numba as nb
+import numpy as np
+
+from eko import constants
+
+from ....harmonics import w1
+
+# from ....harmonics import cache as c
+
+
+@nb.njit(cache=True)
+def gamma_qq(N, cache, is_singlet=None):
+    r"""Compute the LO quark-quark anomalous dimension.
+
+    Implements Eqn. (B.3) from :cite:`Mitov:2006wy`.
+
+    Parameters
+    ----------
+    N : complex
+        Mellin moment
+    cache : numpy.ndarray
+        Harmonic sum cache
+    is_singlet : boolean
+        True for singlet, False for non-singlet, None otherwise
+
+    Returns
+    -------
+    gamma_qq : complex
+        LO quark-quark anomalous dimension
+        :math:`\gamma_{qq}^{(0)}(N)`
+
+    """
+    s1 = w1.S1(N)  # c.get(c.S1, cache, N, is_singlet)
```

Comment:
If you wish you can immediately step into #202 , i.e. base this branch on top of the other

Reply:
I have eliminated the conflicts from this branch for the moment, perhaps I should eliminate them from #202 as well before this step